src/Files.App/UserControls/Widgets/RecentFilesWidget.xaml.cs

```diff
@@ -30,5 +31,54 @@ private void RecentFilesListView_RightTapped(object sender, RightTappedRoutedEve
 		{
 			ViewModel.BuildItemContextMenu(e.OriginalSource, e);
 		}
+
+		private void AddDragAndDropSupport()
+		{
+			RecentFilesListView.CanDragItems = true;
+			// Registering event handlers for drag
+			RecentFilesListView.DragItemsStarting += RecentFilesListView_DragItemsStarting;
```

Comment:
It looks like this is also registered in the XAML, can this line be removed?

Reply:
That's correct RecentFilesListView.DragItemsStarting += RecentFilesListView_DragItemsStarting; can be removed

Reply:
also CanDragItems can be added to XAML so no need of a function called AddDragAndDropSupport, Thanks for pointing it out